README.md

```diff
@@ -1253,6 +1253,13 @@ Other Style Guides
     class Foo {
       bar() { return 2; }
     }
+
+    // good
+    class Foo {
+      increase(x) { return x+1; }
+      // overloading is allowed
+      increase(x, y) { return x+y; }
+    }
```

Comment:
this isn't allowed, because it's not a feature in JS. The second one will overwrite the first one, and so the duplicate class member will obscure a bug.

Reply:
@ljharb Yeah you are right! I was thinking about TypeScript in my head when I wrote that I guess.

Reply:
In that case, the overloads don't have a function body, and you' want to use the `@typescript-eslint/eslint-plugin`'s rules instead of the core rules.